### PR TITLE
Update ITI-55.html

### PR DESCRIPTION
**Pull request by Ben-Levy-62**

### CODE REVIEW

**Comment by JohnMoehrke** [comment]
partial address #491

**Review by dbabato — CHANGES_REQUESTED** [review]
see comments

File: ITI/TF/Volume2/ITI-55.html [hunk 1]
```diff
@@ -936,6 +938,8 @@ <h5 id="3.55.4.1.3">3.55.4.1.3 Expected Actions</h5>
...
       <p>Handling of phonetic issues, alternate spellings, upper and lower case, accented characters, etc., if deemed appropriate, is to be supported by the Responding Gateway rather than by the Initiating Gateway. The Responding Gateway shall return any matches to the query parameters that reflect a high degree of match, after consideration of all policy constraints; IHE does not further specify matching requirements, except as already discussed in the LivingSubjectName parameter description.</p>
+      <h6 id="3.55.4.1.3.2">3.55.4.1.3.2 Targeted Community Processing</h6>
+      <p>If the Responding Gateway serves results from multiple communities, and the Initiating Gateway specified a target community in which to search results as described in section 3.55.4.1.2.5, then the Responding Gateway SHOULD search only the community identified by the homeCommunityId of the receiver device’s Organization. If the homeCommunityId is not one that is served by the Responding Gateway, then it SHOULD return an error as specified in Case 5 of section 3.55.4.2.3.</p>
```
[the remark this answers is not in the document]
Comment (JohnMoehrke) [reply ITI/TF/Volume2/ITI-55.html]: yes. I fixed

File: ITI/TF/Volume2/ITI-55.html [hunk 2]
```diff
@@ -1413,6 +1417,7 @@ <h6 id="3.55.4.2.2.4">3.55.4.2.2.4 Specifying homeCommunityId in Response</h6>
         </li>
       </ul>
       <p>The homeCommunityId is specified as the id element within the assignedEntity of the custodian of the RegistrationEvent. The id element designating the homeCommunityId shall have only the root element, the contents of which is the homeCommunityId.</p>
+      <p>If the Initiating Gateway specified a target community as specified in section 3.55.4.1.2.5, then the Responding Gateway SHOULD include only RegistrationEvents with homeCommunityIds that match the specified target community. However, the Initiating Gateway SHALL NOT rely on this behavior, and SHALL ensure that it is inspecting the homeCommunityId of each RegistrationEvent returned for relevance to its query.</p>
```
[the remark this answers is not in the document]
Comment (JohnMoehrke) [reply ITI/TF/Volume2/ITI-55.html]: yes, I fixed

**Review by dbabato — APPROVED** [review]

**Comment by dbabato** [comment]
Reviewed and found as Okay

**Pull request merged**